pkg/server/server_flows_test.go

```diff
@@ -259,6 +259,16 @@ func TestLokiFiltering(t *testing.T) {
 		Loki: config.Loki{
 			URL:    lokiSvc.URL,
 			Labels: []string{"SrcK8S_Namespace", "SrcK8S_OwnerName", "DstK8S_Namespace", "DstK8S_OwnerName", "FlowDirection"},
+			FieldsType: map[string]string{
+				"Proto":   "number",
+				"Port":    "number",
```

Comment:
I need to figure out why we need `Port` here as it's not a real field. We probably remove the Src / Dst part to catch both